flaskapp/routes.py

```diff
@@ -62,16 +63,12 @@ def logout():
 @app.route("/account", methods=['GET', 'POST'])
 @login_required
 def account():
-    form = UpdateAccountForm()
+    form = UpdateAccountForm(obj=current_user)
```

Comment:
this pre-populates the form, so you don't have to do the
`form.username.data = current_user.username`, as long as the attribute in the form and the model match perfectly, it just works.